python/py-jupyterthemes/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-jupyterthemes
+version             0.20.0
+revision            0
+
+categories-append   devel science
+license             MIT
+maintainers         nomaintainer
+
+description         Select and install a Jupyter notebook theme
+long_description    ${description}
+
+homepage            https://github.com/dunovank/jupyter-themes
+
+checksums           rmd160  ed6b73e604f8ac4c052bbc0cd8d26cc14d741641 \
+                    sha256  2a8ebc0c84b212ab99b9f1757fc0582a3f53930d3a75b2492d91a7c8b36ab41e \
+                    size    6719959
+
+python.versions     39 310
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-ipython \
+                    port:py${python.version}-jupyter_core \
+                    port:py${python.version}-lesscpy \
+                    port:py${python.version}-matplotlib \
+                    port:py${python.version}-notebook
+
+    livecheck.type  none
```

Comment:
this is part of the `python` PG now; as long as there are no multiple versions (e.g., for older Python versions) defined in the Portfile the livecheck is subports will be set to "none" automatically.